sdk/iot/Azure.Iot.Hub.Service/src/ModulesClient.cs

```diff
@@ -294,6 +491,60 @@ public virtual Response<TwinData> UpdateTwin(TwinData twinUpdate, IfMatchPrecond
             return _twinClient.UpdateModuleTwin(twinUpdate.DeviceId, twinUpdate.ModuleId, twinUpdate, ifMatchHeaderValue, cancellationToken);
         }
 
+        /// <summary>
+        /// Update multiple modules' twins. A maximum of 100 updates can be done per call, and each operation must be done on a different module twin. For larger scale operations, consider using <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities">IoT Hub jobs</see>.
+        /// </summary>
+        /// <param name="twinUpdates">The new twins to replace the twins on existing devices.</param>
```

Comment:
Here as well, we've had asks from customers to add explanations on what fields can be updated through these APIs.

I think we should link to some service document which specifies these restrictions, and then it can be on the service team to update them if and when they change the functionality.

Reply:
I'd like the approach too, but there is no doc like that right now. We can circle back on this ask when we talk to the service folks, but I don't see a reason to hold up this pr for now